projects/angular/src/data/datagrid/_datagrid.clarity.scss

```diff
@@ -1511,6 +1568,15 @@
             datagrid-variables.$clr-datagrid-column-toggle-text-hover-color,
             variables.$clr-use-custom-properties
           );
+          cds-icon,
+          clr-icon {
+            @include mixins.css-var(
+              color,
+              clr-datagrid-column-toggle-text-hover-color,
```

Comment:
Same here. Isn't `toggle-icon-color` better for this case?

Reply:
corrected in commit bf22d26